### PR TITLE
Add comment sorting to post page

### PR DESCRIPTION
**Pull request by matthewdias**

### CODE REVIEW

File: app/components/stream-feed/items/post/comments.js [hunk 1]
```diff
@@ -66,10 +66,10 @@ export default Component.extend(Pagination, {
...
     } else {
       // don't reload if the we have received attrs but the post hasn't changed
-      if (get(this, 'post.id') === get(this, 'postIdWas')) {
-        return;
-      }
-      set(this, 'postIdWas', get(this, 'post.id'));
+      // if (get(this, 'post.id') === get(this, 'postIdWas')) {
+      //   return;
+      // }
+      // set(this, 'postIdWas', get(this, 'post.id'));
```
Comment (matthewdias) [inline app/components/stream-feed/items/post/comments.js]: @vevix what is the purpose of this check?

Comment (trmcnvn) [reply app/components/stream-feed/items/post/comments.js]: as per the comment: So we don't re-perform the request when params change.

Creating a comment will increase the post's comment count, which will proxy down to this comment hook and reload the comments.

It's kinda bad, but the entire feeds need to be reworked.

**Review by trmcnvn — CHANGES_REQUESTED** [review]

**Comment by JoshFabian** [comment]
@vevix @matthewdias any blockers in terms of getting this merged atm?

**Comment by trmcnvn** [comment]
Yeah, small blocker. (See my review thread)

**Comment by JoshFabian** [comment]
@vevix is this something you can take care of? I don't think @matthewdias will be back online in the next 12 or so hours.

**Comment by trmcnvn** [comment]
Yeah, I will if he doesn't return before I get around to it.

**Pull request merged**